build.sbt

```diff
@@ -126,4 +127,15 @@ def playProject(projectName: String, port: Int): Project =
         (packageBin in Debian).value -> s"${name.value}/${name.value}.deb",
         file(s"$projectName/conf/riff-raff.yaml") -> "riff-raff.yaml"
       )
-    ))
\ No newline at end of file
+    ))
+
+val testSettings = Seq(
+  testOptions in Test += Tests.Setup(_ => {
+
+    println(s"Launching docker container with ES")
+    s"docker-compose up -d".!
```

Comment:
This'll run all the services defined [`docker-compose.yml`](https://github.com/guardian/grid/blob/master/docker-compose.yml). I don't think we'd need the imgops container in CI, so we could run `docker-compose up -d elasticsearch` instead to only start an elasticsearch container.